production/helm/promtail/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: promtail
-version: 0.12.2
+version: 0.12.3
```

Comment:
you need to also bump the loki-stack chart.

Reply:
Reverting version until after helm changes.